net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
Yes.  Just think it could be better if the check is to be done only once.

Well, this is implementation details and you are the maintainer :)  Thanks for the efforts and bearing with me being pedantic or whatever in such things